### PR TITLE
docs(global): update README.md

### LINKED ISSUE #45: [DOCS] Fix API and UI setup links in global readme
Currently `ui setup` and `api setup` both point to `ui` README.

Action Items:
- [ ]  Update the `api setup` link to point to API's README.
- [ ]  Clicking on the link should land and scroll to the README instead of at the top of the screen.

### PR DESCRIPTION
**Pull request by PrajwalBorkar**
Fixes API readme link and UI setup links in global readme
Fixes #45

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,8 +15,8 @@ merchandise, etc. for all the gymkhana clubs in our institute.
 
 - Fork and clone the repository.
 - Add remote upstream `git remote add upstream https://github.com/BitByte-TPC/gymkhana.git`
-- [ui setup](https://github.com/BitByte-TPC/gymkhana/tree/master/ui)
-- [api setup](https://github.com/BitByte-TPC/gymkhana/tree/master/ui)
+- [ui setup](https://github.com/BitByte-TPC/gymkhana/tree/master/ui#readme)
+- [api setup](https://github.com/BitByte-TPC/gymkhana/tree/master/ui#readme)
```
Comment (Aksh-Bansal-dev) [inline README.md:19]:
```suggestion
- [api setup](https://github.com/BitByte-TPC/gymkhana/tree/master/api#readme)
```

**Review by Aksh-Bansal-dev — CHANGES_REQUESTED** [review]

**Review by Aksh-Bansal-dev — DISMISSED** [review]
LGTM.

**Comment by Aksh-Bansal-dev** [comment]
Update the commit message to follow the [commit guidelines](https://github.com/BitByte-TPC/gymkhana/blob/master/CONTRIBUTING.md#standard-commit-messages) and also try to write a more meaningful commit message.

**Review by varisha-025 — APPROVED** [review]
LGTM.

**Comment by Samy-33** [comment]
@PrajwalBorkar, use [git rebase](https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase) to squash the two commits you've created into one with a message conforming to the guidelines. And then force push using `git push -f`

**Comment by PrajwalBorkar** [comment]
> @PrajwalBorkar, use [git rebase](https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase) to squash the two commits you've created into one with a message conforming to the guidelines. And then force push using `git push -f`

Yeah done. Please review

**Review by Aksh-Bansal-dev — APPROVED** [review]
LGTM.

**Pull request merged**